packages/gasket-plugin-express/lib/create.js

```diff
@@ -24,12 +23,15 @@ module.exports = async function create(gasket, context) {
 
     if ('typescript' in context && context.typescript) {
       context.files.add(`${generatorDir}/**/!(*.js)`);
+      context.gasketConfig
+        .addImport('{ routes }', './routes/index');
```

Comment:
Not sure if you tested this with Typescript, this likely won't work due to ESM errors. The current `v7`:
```js
    context.gasketConfig
      .addImport('{ routes }', './routes/index.js')
      .injectValue('express.routes', 'routes');
```

This is by design. We use `.js` imports for both TS and JS.